docs/using-the-jdbc-driver/RdsProxy.md

```diff
@@ -0,0 +1,42 @@
+| Flexibility   | If you already have a community JDBC driver fully integrated in your workflow, such as the PostgreSQL JDBC driver or the MySQL Connector/J. The AWS Advanced JDBC Driver allows customers to continue using their existing community drivers in addition to having the AWS JDBC Driver fully exploit failover behavior by maintaining a cache of the Aurora cluster topology and each DB instance's role. | The RDS Proxy provides an additional point of failure to the databases and can act as an additional layer of security between client applications and the underlying database. However the proxies require network connection and may introduce extra complexity during local or offline development.                                                              |
+| Custom DNS    | The AWS JDBC Driver supports custom DNS.                                                                                                                                                                                                                                                                                                                                                                  | The RDS Proxy does not support custom DNS.                                                                                                                                                                                                                                                                                                                         |
+| Configuration | The AWS JDBC Driver allows you to fine tune the failover process with parameters like `failoverTimeoutMs` and `failoverReaderConnectTimeoutMs`. For more information on failover parameters, see the [FailoverConfigurationGuide](./FailoverConfigurationGuide.md).                                                                                                                                       | Each RDS Proxy handles connections to an RDS instance or an Aurora cluster. To enable failover for multiple clusters you would have to configure RDS Proxy for each cluster. While the RDS Proxy provides some configurations, such as the database authentication method, it does not offer the same level of failover configuration as the AWS JDBC Driver does. |
+
+## Differences in Features
+
+| Feature                        | AWS JDBC Driver                                                                               | RDS Proxy                                                                                                                                                                                                                                                                                                                                              |
```

Comment:
consistency: I notice that some sentences end with a period and some do not in the table. Could we re-review and make this consistent with the period?